apps/src/javalab/Javalab.js

```diff
@@ -41,6 +46,8 @@ Javalab.prototype.init = function(config) {
 
   this.skin = config.skin;
   this.level = config.level;
+  this.suppliedFilesVersionId = queryParams('version');
+  this.initialFilesVersionId = this.suppliedFilesVersionId;
```

Comment:
I'm not super familiar with what these two lines do. Mind adding a comment?

Reply:
did some refactoring of this--we didn't actually need to track initialFilesVersionId, so changed suppliedFilesVersionId to a const below and added a comment as to what it does.

Reply:
👍